src/components/visual-flow/nodes/authentication-option-node.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { FunctionComponent, ReactElement } from "react";
+import { Handle, Position } from "react-flow-renderer";
+import Jwt from "../../../assets/jwt.png";
+import { Facebook, Fido, Google, Instagram, Totp, Twitter } from "../../core";
+
+/**
+ * Authentication Option Node component.
+ *
+ * @return {React.ReactElement}
+ */
+// eslint-disable-next-line react/prop-types,@typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
What is the reason for `@ts-ignore` here. It is usually an anti pattern.

Reply:
Fixed with [34d0251](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/8/commits/34d0251e3a545bb19ec6307d398fe375a97d2b52)